api/test/application/user/user-register-post.js

```diff
@@ -122,14 +117,159 @@ describe(`POST /api/invite`, () => {
     expect(user_profile.phone_number).to.equal(payload.phone_number);
   });
 
+  /*it(`returns 400 with missing payload`, async () => {
+    const queryString = querystring.stringify(getQueryParams());
+    const res = await server.inject({
+      method: "POST",
+      url: `/user/register?${queryString}`,
+    });
+    expect(res.statusCode).to.equal(400);
+
+  });*/
```

Comment:
Can we get rid of this commented out code?

Reply:
The test now passes and works as expected with the most recent change to `mixed-validation.js`